### DISCUSSION TITLE
UpdatedAt column in mongoDB is not updating on direct update on database

### DISCUSSION THREAD

**Discussion by vinaykharayat** [prisma/prisma#21067] [Q&A] [answered]
### Question

If I update the db using prisma, updatedAt column is updating its value, but when I am modifying data directly in mongoDb, updatedAt is not updating.

I am not sure how prisma handle this but in RDBMS we create triggers and it doesn't matter how we change row value, updatedAt update its time.

### How to reproduce (optional)
1. Create a schema
2. Insert value in db
3. Directly modify the value in DB
4. updatedAt is not updating

### Expected behavior (optional)

Should update updatedAt if directly modifying db values.

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
model ModelName {
  id        String         @id @default(auto()) @map("_id") @db.ObjectId
  user      Users?         @relation(fields: [userId], references: [id])
  userId    String?        @db.ObjectId
  rating    Int
  message   String
  createdAt DateTime       @default(now())
  updatedAt DateTime?      @updatedAt
}
```

- OS: 
- Database: MongoDB
- Node.js version:v18.16.0

- Run `prisma -v` to see your Prisma version and paste it
4.16.2

**Comment by janpio** [comment, marked as answer]
`@updatedAt` is implemented on the Prisma side, not on the database - especially for MongoDB. You will have to deal with the updates manually when you are using other database Clients.
